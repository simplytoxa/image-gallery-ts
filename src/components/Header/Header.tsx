import React from 'react';
import './Header.css';

const Header = props => {
  return (
    <header className="Header">
      {props.children}
    </header>
  );
};

export default Header;
