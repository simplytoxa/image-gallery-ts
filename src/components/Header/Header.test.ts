import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import Header from './Header';

describe('<Header>', () => {
  const wrapper = shallow(
    <Header>Header</Header>
  );
  it('should render a header', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
