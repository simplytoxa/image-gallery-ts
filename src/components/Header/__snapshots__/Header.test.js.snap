// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> should render a header 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header>
    Header
</Header>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Header",
      "className": "Header",
    },
    "ref": null,
    "rendered": "Header",
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Header",
        "className": "Header",
      },
      "ref": null,
      "rendered": "Header",
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
